Sources/Testing/ExitTests/ExitTest.swift

```diff
@@ -679,37 +762,50 @@ extension ExitTest {
         var backChannelWriteEnd: FileHandle!
         try FileHandle.makePipe(readEnd: &backChannelReadEnd, writeEnd: &backChannelWriteEnd)
 
-        // Let the child process know how to find the back channel by setting a
-        // known environment variable to the corresponding file descriptor
-        // (HANDLE on Windows.)
-        var backChannelEnvironmentVariable: String?
-#if SWT_TARGET_OS_APPLE || os(Linux) || os(FreeBSD) || os(OpenBSD)
-        backChannelEnvironmentVariable = backChannelWriteEnd.withUnsafePOSIXFileDescriptor { fd in
-          fd.map(String.init(describing:))
-        }
-#elseif os(Windows)
-        backChannelEnvironmentVariable = backChannelWriteEnd.withUnsafeWindowsHANDLE { handle in
-          handle.flatMap { String(describing: UInt(bitPattern: $0)) }
-        }
-#else
-#warning("Platform-specific implementation missing: back-channel pipe unavailable")
-#endif
-        if let backChannelEnvironmentVariable {
+        // Create another pipe to send captured values (and possibly other state
+        // in the future) to the child process.
```

Comment:
Is this the only "from parent to child" pipe, or was there another one previously and this is a second? (I know there's a pipe in the opposite direction, just asking whether this is the only one which goes from parent to "exit test" child.)

Reply:
This is the only from-parent-to-child pipe. The existing pipe goes the other direction, and pipes are not bidirectional. (In the future, if we have other data to send from parent to child, I'd use this pipe for all such data, but for now we only need the one thing.